src/modules/commentPopUp.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable guard-for-in */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.

Reply:
Thank you for pointing this out, it has been successfully removed.

https://github.com/iamchristianani/js-capstone/blob/febf719dac18cfc2152e1eb6f39eb294d7f304a2/src/modules/commentPopUp.js#L1-L3